### ISSUE TITLE
New channel get a private channel

### ISSUE THREAD

**Issue by QueenGit** [LN-Zap/zap-iOS#241]
**Describe the bug**
I open a new channel and its automatically a privat one. There should be a public one by default, like it was in there early versions.

**Your environment**
- Version 0.4.0

**Comment by ottosuess** [comment]
we open private channels because we expect most people will run lnd on their phones.

phones won't be able to route so private channels are the way to go 🤔

**Comment by QueenGit** [comment]
That's a good point, but I use Zap to mange my LND node. There would it be nice if I could open a public one.

**Comment by JimmyMow** [comment]
Makes sense to allow a public/private toggle for remote node users maybe

**Comment by ottosuess** [comment]
or maybe open public channels for remote connections and private channels for lnd running on the device?

**Comment by mrfelton** [comment]
In the desktop app, we default to private channels for Neutrino nodes and public channel's for remote nodes. For remote node users we also display a `private` toggle to allow them to override our default and make their channel private if they like.

**Comment by QueenGit** [comment]
> In the desktop app, we default to private channels for Neutrino nodes and public channel's for remote nodes. For remote node users we also display a `private` toggle to allow them to override our default and make their channel private if they like.

That would be a good way for the iOS version to, I think.

**Comment by k0gen** [comment]
> Makes sense to allow a public/private toggle for remote node users maybe

Yes. It can stay private by default but at least make an public/private toggle option for remote node users.

**Comment by jheathco** [comment]
I also agree there needs to be an option for this.
